crates/net/discv4/src/proto.rs

```diff
@@ -215,33 +215,132 @@ impl NodeEndpoint {
 }
 
 /// A [FindNode packet](https://github.com/ethereum/devp2p/blob/master/discv4.md#findnode-packet-0x03).
-#[derive(Clone, Copy, Debug, Eq, PartialEq, RlpEncodable, RlpDecodable)]
+#[derive(Clone, Copy, Debug, Eq, PartialEq, RlpEncodable)]
 pub struct FindNode {
```

Comment:
is this the same as #[rlp(trailing)]

see also:

https://github.com/paradigmxyz/reth/blob/0aa7d4d05ed5565545002655ddc94083e057977e/crates/ethereum-forks/src/forkid.rs#L125-L127